### PR TITLE
Build 2.1.0-1

### PR DESCRIPTION
**Pull request by sgillies**
Planet 2.1.0 requires click < 8.1.4. That will be resolved in version 2.1.1. For now, we'll require that package tests run with click 8.1.3.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sgillies** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
> Planet 2.1.0 requires click < 8.1.4. That will be resolved in version 2.1.1. For now, we'll require that package tests run with click 8.1.3.

@sgillies I can do that with a repo data patch in a sec. That way we avoid a new build. Hold on...

**Comment by ocefpaf** [comment]
It turns our the build number 0 never happened due to a CI problem after the merger. See https://github.com/conda-forge/planet-feedstock/runs/15143110083

Investigating...

**Comment by sgillies** [comment]
@ocefpaf yes, I know the problem: a click version mismatch between the package and the build environment. That's why I want to pin click in meta.yaml.

**Comment by ocefpaf** [comment]
See https://github.com/conda-forge/planet-feedstock/pull/49#issuecomment-1640743025

TL;DR it should be safe to merge this one.

**Comment by sgillies** [comment]
Thanks @ocefpaf !
